pkg/providers/aws/cluster_network_provider.go

```diff
@@ -50,22 +50,20 @@ import (
 )
 
 const (
-	DefaultRHMIVpcNameTagValue           = "RHMI Cloud Resource VPC"
-	DefaultRHMISubnetNameTagValue        = "RHMI Cloud Resource Subnet"
-	DefaultRHMISecurityGroupNameTagValue = "RHMI Cloud Resource Security Group"
-	DefaultRHMIVpcRouteTableNameTagValue = "RHMI Cloud Resource Route Table"
-	defaultNumberOfExpectedSubnets       = 2
-	clusterOwnedTagKeyPrefix             = "kubernetes.io/cluster/"
-	clusterOwnedTagValue                 = "owned"
-	clusterSharedTagValue                = "shared"
-	//network peering
-	tagVpcPeeringNameValue = "RHMI Cloud Resource Peering Connection"
+	clusterOwnedTagKeyPrefix                = "kubernetes.io/cluster/"
```

Comment:
@cecobask all looks nice here for this file, refactoring and improvements. Just could I ask - while comparing with master, can't understand when STS was removed (maybe in another PR that this branch is based ?) or vice-a-versa, STS is new (need look), but if so maybe need rebase. Rebase anyway seems needed, as comparing with master give more diff, as I see. But changes that done look very nice

Reply:
@valerymo I based this branch off from master and at that time STS features weren't merged into master. I will definitely need to do a rebase as the STS feature branch was merged yesterday.